source/upgrade/important-upgrade-notes.rst

```diff
@@ -21,6 +21,13 @@ Important Upgrade Notes
 +----------------------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | If you’re upgrading from a version earlier than... | Then...                                                                                                                                                          |
 +====================================================+==================================================================================================================================================================+
+| v6.4                                               | A new schema migration system has been introduced. Before updating the app to this version, it is strongly recommended to have a database backup. The new        |
+|                                                    | migration system will run through all existing migrations to record them to a new table. This will only happen for the first run in order to migrate the         |
+|                                                    | application to the new system. The table where the migration information is stored is ``db_migrations``. Additionally, a ``db_lock`` table is used to prevent    |
```

Comment:
```suggestion
|                                                    | application to the new system. The table where the migration information is stored is called ``db_migrations``. Additionally, a ``db_lock`` table is used to prevent    |
```